awx/main/tasks.py

```diff
@@ -1102,6 +1102,8 @@ def run(self, pk, **kwargs):
         self.instance = self.update_model(pk, status='running',
                                           start_args='')  # blank field to remove encrypted passwords
 
+        self.instance.send_notification_templates("running")
```

Comment:
Yep, I think this is a reasonable place to do this.  @AlanCoding @chrismeyersfsu you have any opinions?

Reply:
Had to look at the code for `send_notification_templates`, but it calls `.delay` on the relevant task to _actually_ send the notification templates. So that is what we want, because it makes it congruent with `websocket_emit_status`, where we create some _internal_ network traffic within the cluster. That looks good.

I would slightly prefer that both of these methods move just a few lines down to inside of the `try:` block. For anything that isn't trivial variable initialization, I would prefer for errors to be surfaced in the job traceback.

Reply:
@beeankha I agree with @AlanCoding.  Let's move this function call lower into the `try:` block below.  Maybe right before the `self.pre_run_hook()` call?

Reply:
Done!